Code/GraphMol/MolDraw2D/MolDraw2DUtils.cpp

```diff
@@ -400,5 +401,46 @@ void contourAndDrawGaussians(MolDraw2D &drawer,
   contourAndDrawGrid(drawer, grid.get(), xcoords, ycoords, nContours, levels,
                      paramsCopy);
 };
+
+// ****************************************************************************
+void reapplyMolBlockWedging(ROMol &mol) {
```

Comment:
I think this function is broadly useful - I've done a couple different Python implementations for other projects and doing a C++ implementation has been on my ToDo list for a while.
It would be better to have this in `GraphMol/FileParsers/MolFileStereochem.{cpp,h}`

Reply:
Where would you like the test case?  At the moment it's in GraphMol/MolDraw2D/catch_tests.cpp and checks where the triangle is in the drawing.

Reply:
I will expose it to Python as well.  Can't think why I forgot to do that.

Reply:
> Where would you like the test case?  At the moment it's in GraphMol/MolDraw2D/catch_tests.cpp and checks where the triangle is in the drawing.

There should be a test case in the fileparsers catch test file I think